lutris/services/gog.py

```diff
@@ -265,17 +282,16 @@ def new_from_gog_game(cls, gog_game):
         return service_game
 
     @classmethod
-    def get_banner(cls, gog_game):
+    def get_banner(cls, gog_game, size="medium"):
         """Return the path to the game banner.
         Downloads the banner if not present.
         """
-        # Are there other formats?
-        image_url = "https:%s_prof_game_100x60.jpg" % gog_game["image"]
-        image_hash = gog_game["image"].split("/")[-1]
-        cache_dir = os.path.join(settings.CACHE_DIR, "gog/banners/small/")
+        image_url = "https:%s%s" % (gog_game["image"], GOG_BANNER_FORMATS[size])
```

Comment:
Shouldn't there be // after https:, like https://%s%s?

Reply:
no it's included by GOG